File: crm/src/main/java/com/legion/test/TestController.java
package com.legion.test;

import org.springframework.http.HttpStatus;
import org.springframework.http.ResponseEntity;
import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;

@Controller
@RequestMapping("/s2s")
public class TestController {

    @GetMapping
    public ResponseEntity<String> test() {
        return new ResponseEntity<>("Hello crm!", HttpStatus.OK);
    }
}

File: crm/src/main/kotlin/com/legion/test/TestController.kt
package com.legion.test

import org.springframework.beans.factory.annotation.Autowired
import org.springframework.http.HttpStatus
import org.springframework.http.ResponseEntity
import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.GetMapping

@Controller
class TestController @Autowired constructor(

) {

    @GetMapping("/test")
    fun test(): ResponseEntity<String> {
        return ResponseEntity("Hello crm and rest template", HttpStatus.OK)
    }
}